src/main/kotlin/no/nav/pam/feed/ad/InputPayload.kt

```diff
@@ -1,32 +1,19 @@
 package no.nav.pam.feed.ad
 
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import com.fasterxml.jackson.annotation.JsonProperty
-import com.fasterxml.jackson.core.JsonParser
-import com.fasterxml.jackson.databind.DeserializationContext
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize
-import com.fasterxml.jackson.databind.deser.std.StdDeserializer
 import java.time.ZonedDateTime
 
 
-@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
👍

Reply:
Burde også se om vi kan gjøre noe tilsvarende for de andre @Json* annontasjonene